.expeditor/config.yml

```diff
@@ -44,8 +44,3 @@ subscriptions:
     actions:
       - built_in:rollover_changelog
       - built_in:publish_rubygems
-
-pipelines:
-  - verify:
```

Comment:
Any reason to remove the verify pipeline for this?

Reply:
As discussed I have added it again